html/search.html

```diff
@@ -1,25 +1,58 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="/css/search.css">
-    <link rel="preconnect" href="https://fonts.gstatic.com">
-<link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro&display=swap" rel="stylesheet">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="../css/search.css" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro&display=swap"
+      rel="stylesheet"
+    />
     <title>Search</title>
-</head>
-<body>
-    <header><h1>Covid App</h1></header>
-   
-    
+  </head>
+
+  <body>
+    <header>
+      <h1>Covid App</h1>
+    </header>
+
+    <section class="Search-section">
+      <input
+        list="countries"
+        id="selectedCountry"
+        placeholder="     search..."
+      />
+      <datalist id="countries"></datalist>
+      <button id="search-button">Search</button>
+    </section>
```

Comment:
The results are a little bit hard to read because of the background
 its need more attention to  design colours  and measurements

![Screenshot from 2021-04-15 17-46-24](https://user-images.githubusercontent.com/47992412/114889481-d54b1680-9e12-11eb-8ab9-441bb18ba5f8.png)

![Screenshot from 2021-04-15 17-49-35](https://user-images.githubusercontent.com/47992412/114889761-0e838680-9e13-11eb-9a81-709b7700c3a8.png)